### PR TITLE
Fixing conda environment and installation in tests

### PR DESCRIPTION
**Pull request by sylvchev**
This PR aims at solving dependencies issues with conda and requirements files. Issues with poetry will be addressed in another PR.

### CODE REVIEW

**Comment by sylvchev** [comment]
Removed conda environment that was taking too long to install due to dependencies resolution (between 5 minutes to never ending).

File: setup.py [hunk 1]
```diff
@@ -34,9 +34,12 @@
     license='BSD 3-Clause',
 
     install_requires=['mne', 'numpy', 'pandas', 'scipy', 'matplotlib', 'h5py', 'skorch',
-                      'einops'],
+                      'torch', 'einops', 'joblib'],
     extras_require={
-        'moabb': ["moabb"]
+        'moabb': ['moabb'],
+        'tests': ['pytest','pytest-cov','codecov'],
```
Comment (bruAristimunha) [inline setup.py:40]:
```suggestion
        'tests': ['pytest', 'pytest-cov', 'codecov'],
```

**Review by bruAristimunha — COMMENTED** [review]
Amazing @sylvchev, can you please update the whats_new file and the documentation?

**Review by bruAristimunha — APPROVED** [review]

**Comment by sylvchev** [comment]
LGTM @bruAristimunha !

**Pull request merged**